index.js

```diff
@@ -141,44 +186,63 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+
+let ticket = []
+function takeATicketNumber(line) {
+  ticket.push(line.length)
+  console.log(`Welcome. You are ticket number ${ticket.length}`)
+  line.push(line.length + 1)
+}
 
```

Comment:
This approach works! This can be simplified by replacing the `ticket` array with another data type. You're currently only using the `ticket` array for its length and not because of any value it stores, so this beats the purpose of using an array. What other data type could you change `ticket` to simplify this?